cytodynamics-test-container/src/test/java/com/linkedin/cytodynamics/test/TestDynamicLoad.java

```diff
@@ -318,8 +404,10 @@ public void testSecurity() throws Exception {
           .anIsolatingLoader()
           .withOriginRestriction(onlyTmpOriginRestriction)
           .withClasspath(Collections.singletonList(getTestJarUri("a")))
-          .withIsolationLevel(IsolationLevel.FULL)
-          .addWhitelistedClassPattern("java.*")
+          .addParentRelationship(ParentRelationshipBuilder.builder()
+              .withIsolationLevel(IsolationLevel.FULL)
+              .addWhitelistedClassPattern("java.*")
+              .build())
```

Comment:
I'm wondering if we might want to support a closure syntax for `addParentRelationship()`, where a builder is provided as an input to the closure, and `build()` is automatically called on the returned (modified) builder after running the closure.

Calling this new overload would look like this:

```
          .addParentRelationship(builder -> builder
              .withIsolationLevel(IsolationLevel.FULL)
              .addWhitelistedClassPattern("java.*"))
```

This is just syntactic sugar to eliminate a tiny bit of boilerplate and doesn't really change much. Feel free to adopt or disregard.

Reply:
Hm, interesting suggestion. I think I would still prefer passing the built `ParentRelationship`. I think it is slightly more straightforward, even if it requires extra boilerplate.